prql-compiler/src/parser/chumsky.rs

```diff
@@ -78,3 +85,35 @@ pub fn ident() -> impl Parser<char, Ident, Error = Simple<char>> {
         .chain(just('.').ignore_then(ident_part().or(star)).repeated())
         .map(Ident::from_path::<String>)
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    use insta::assert_yaml_snapshot;
+    #[test]
+    fn test_date() {
+        let date = just('@').ignore_then(
+            digits(4)
+                .then_ignore(just('-'))
+                .then(digits(2))
+                .then_ignore(just('-'))
+                .then(digits(2)),
+        );
```

Comment:
This is arguably more readable than PEST derivations.

Reply:
Yes it's not too bad!